docs/installation.md

```diff
@@ -0,0 +1,92 @@
+### Content
+As shown below, the release tarball contains:
+1) `cri-containerd`: cri-containerd binary.
+2) `containerd`, `containerd-shim`, `containerd-stress`, `ctr`: binaries for containerd.
+3) `runc`: runc binary.
+4) `crictl`: command line tools for CRI container runtime.
+5) `containerd.service`, `cri-containerd.service`: Systemd units for cri-containerd and containerd.
+```console
+$ tar -tf cri-containerd-1.0.0-alpha.0.tar.gz 
+./
+./usr/
+./usr/local/
+./usr/local/sbin/
+./usr/local/sbin/runc
+./usr/local/bin/
+./usr/local/bin/crictl
+./usr/local/bin/containerd
+./usr/local/bin/containerd-stress
+./usr/local/bin/containerd-shim
+./usr/local/bin/ctr
+./usr/local/bin/cri-containerd
+./etc/
+./etc/systemd/
+./etc/systemd/system/
+./etc/systemd/system/containerd.service
+./etc/systemd/system/cri-containerd.service
+```
+### Binary Information
+Information about the binaries in the release tarball:
+
+|           Binary Name          |      Support      |   OS  | Architecture |
+|:------------------------------:|:-----------------:|:-----:|:------------:|
+|          cri-containerd        | seccomp, apparmor | linux |     amd64    |
+|               runc             | seccomp, apparmor | linux |     amd64    |
+|   containerd/containerd-shim   |   overlay, btrfs  | linux |     amd64    |
+
+If you have other requirements for the binaries, e.g. selinux support, another architecture support etc., you need to build the binaries yourself following [the instructions](../README.md#getting-started-for-developers).
+ 
+### Download
+
+The release tarball could be downloaded from either of the following sources:
+1. Release on github (see [here](https://github.com/kubernetes-incubator/cri-containerd/releases));
+2. Release GCS bucket https://storage.googleapis.com/cri-containerd-release/.
+
+## Step 0: Install Required Libraries
+Install required libraries for seccomp and libapparmor.
+```shell
+sudo apt-get update
+sudo apt-get install libseccomp2
+sudo apt-get install libapparmor
+```
+Note that:
+1) If you are using Ubuntu <=Trusty or Debian <=jessie, a backported version of `libseccomp2` is needed. (See the [trusty-backports](https://packages.ubuntu.com/trusty-backports/libseccomp2) and [jessie-backports](https://packages.debian.org/jessie-backports/libseccomp2)).
+2) If your OS distro doesn't support AppArmor, please skip installing `libapparmor`, and AppArmor will be disabled.
+## Step 1: Download CRI-Containerd Release Tarball
+Download release tarball for the cri-containerd version you want to install from sources listed [above](#download).
+```console
+$ wget https://storage.googleapis.com/cri-containerd-release/cri-containerd-${VERSION}.tar.gz
+```
+## Step 2: Install CRI-Containerd
+If you are using systemd, just simply unpack the tarball to the root directory:
+```shell
+sudo tar -C / -xzf cri-containerd-${VERSION}.tar.gz
+sudo mkdir -p /etc/cni/net.d
```

Comment:
how about /opt/cni/bin/

Reply:
ocicni only needs `/etc/cni/net.d` to be there.

Reply:
i meant we need to mkdir  /opt/cni/bin/ also :)

Reply:
Hm, really? OK, will add it.

Reply:
@Random-Liu : https://github.com/kubernetes-incubator/cri-containerd/blob/eb2ea51e00fcc37696499718617831109fe737bf/contrib/ansible/tasks/binaries.yaml#L8

Reply:
Done.